apache-rat-core/pom.xml

```diff
@@ -126,5 +126,10 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Could we add the dependency into a dependencyManagement block to only have the version in 1 pom.xml and reference it in the other one?

Reply:
Thanks.